### ISSUE TITLE
Feature Request: Use multiple bots to bypass discord message send cap

### ISSUE THREAD

**Issue by BobMcclaw** [NPCat/bad-apple-bot#4]
I see this as viable, only problem would be server lag but hopefully not too high impact

**Comment by Teinc3** [comment]
I Agree. However how many bots would it take to sustain the framerate of the video? It must take a lot.

**Comment by Fish-o** [comment]
thats against the api policy :(


> API Limits.
> Discord sets and enforces limits on your use of the APIs (for example, by limiting the number of API requests that you may make, the number of servers your application is in, or the number of users you may serve), in our sole discretion. You agree to, and will not attempt to circumvent, such limitations documented with each API. 
> 

https://discord.com/developers/docs/policy#api-limits

**Comment by Teinc3** [comment]
Ah, i See. Then, unfortunately, this idea has to be scraped

**Comment by mouncg** [comment]
lmao why don't you just try to pull a edit and send trick
just keep a running count of the ratelimits and when send gets used, edit the previous msg
rinse and repeat?

**Comment by m00nwtchr** [comment]
or webhooks, dunno if that technically is against the rules as well

**Comment by MGDproductions** [comment]
> or webhooks, dunno if that technically is against the rules as well

webhooks also has a limit that is even lower then the bot limit

**Comment by Natemo2625** [comment]
i think editing the previous message would be better tbh

**Comment by m00nwtchr** [comment]
> > or webhooks, dunno if that technically is against the rules as well
> 
> webhooks also has a limit that is even lower then the bot limit

yea but you can make lots of them, but yeh just editing the message would be ideal imo, but i don't think that can be done fast enough for a proper video either

**Comment by Fish-o** [comment]
> > > or webhooks, dunno if that technically is against the rules as well
> > 
> > webhooks also has a limit that is even lower then the bot limit
> 
> yea but you can make lots of them, but yeh just editing the message would be ideal imo, but i don't think that can be done fast enough for a proper video either

yup, also a rate limit on that
